github/ci/prow-deploy/kustom/base/configs/current/config/config.yaml

```diff
@@ -744,6 +744,19 @@ presets:
   - name: pgp-bot-key
     secret:
       secretName: pgp-bot-key
+- labels:
+    preset-project-infra-image-build-linux: "true"
+  env:
+    - name: GIMME_GO_VERSION
+      value: "1.25.1"
```

Comment:
this should be fine as it is a specific preset for the project-infra repo - it shouldn't be used by other repos in the org.